pykeepass/entry.py

```diff
@@ -81,11 +81,14 @@ def _get_string_field(self, key):
         if field is not None:
             return field.text
 
-    def _set_string_field(self, key, value):
+    def _set_string_field(self, key, value, protect=False):
         field = self._xpath('String/Key[text()="{}"]/..'.format(key), first=True)
         if field is not None:
             self._element.remove(field)
-        self._element.append(E.String(E.Key(key), E.Value(value)))
+        if protect:
+            self._element.append(E.String(E.Key(key), E.Value(value, Protected="False")))
```

Comment:
Why `Protected="False"` and not `True`?

Reply:
At the time I made this PR, https://github.com/libkeepass/pykeepass/blob/43c0d64abcbcd55e1f1e05c2e4c5d6ac0527d9ec/pykeepass/entry.py#L52 used to be
```python
E.String(E.Key('Password'), E.Value(password, protected="False"))
```
(see https://github.com/libkeepass/pykeepass/pull/186/commits/daaf7826b0eacaf0af7e867a106d5d1180fd3a81)

Also, as I mentioned it the PR description, when opening a database created by KeePass containing a protected value, it appears to have ` Protected="False"` in the raw XML.

I mimicked this behavior and it worked well, but I don't fully understand how this field works.